src/IssueViz.UnitTests/Helpers/ShowInBrowserServiceTests.cs

```diff
@@ -122,35 +121,15 @@ public void ShowCommunityPage_BrowserOpened()
             browserService.VerifyNoOtherCalls();
         }
 
-        [TestMethod]
-        public void ShowRuleDescription_BrowserOpened()
```

Comment:
No longer required. The method being tested has been removed from the interface.